bindings/README.md

```diff
@@ -17,15 +17,30 @@ In order to generate rust bindings for s2n-tls, you need to have the following i
 Generating rust bindings can be accomplished by running the `generate.sh` script:
 
 ```
-$ ./bindings/rust/generate.sh
+$ ./bindings/rust/extended/generate.sh
 ```
 
 This script generates the low-level bindings in the crate `s2n-tls-sys`, which is used by the `s2n-tls` crate to provide higher-level bindings.
 See [s2n-tls-sys](https://github.com/aws/s2n-tls/blob/main/bindings/rust/s2n-tls-sys/README.md) for more information on `s2n-tls-sys` crate.
 
 ## Minimum Supported Rust Version (MSRV)
 
-`s2n-tls` will maintain a rolling MSRV (minimum supported rust version) policy of at least 6 months. The current s2n-quic version is not guaranteed to build on Rust versions earlier than the MSRV.
+There are three workspaces, with slightly different MSRV policies, to keep broad support for the bindings, while allowing newer MSRV for integrations, like `s2n-tls-hyper`.
```

Comment:
Aren't there two workspaces? standard and extended?

Also nit, but this sentence is kind of running on. Maybe something like this?
```suggestion
There are two rust bindings workspaces that have different MSRV policies. Crates in `standard` maintain a rolling MSRV policy of at least 6 months. Crates in `extended` maintain an older MSRV for increased support.
```